cargo-geiger/src/cli.rs

```diff
@@ -938,7 +942,8 @@ fn resolve_rs_file_deps(
         config: &config,
         spec: vec![],
         target: None,
-        release: false,
+        profile_specified: false,
+        profile_kind: ProfileKind::Release,
```

Comment:
Is this correct?

Reply:
The old flag was the boolean `release` set to false, the new one should be ProfileKind::Dev to match CompileMode::Check here:
https://github.com/anderejd/cargo-geiger/blob/cca096062a726a535ebfacefda328d58535b05e6/cargo-geiger/src/cli.rs#L672-L674

There's definitely a case for switching to Release profile as the default though, but let's do that in a new issue, unrelated to this library upgrade.

Reply:
Of course, how silly of me! I have amended my commit to use `ProfileKind::Dev`.